test/networkClient.test.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import { NetworkClient } from '@subql/network-clients';
-import { SQNetworks } from '@subql/network-clients';
+import { NetworkClient } from '../packages/network-clients';
+import { SQNetworks } from '../packages/network-clients';
 
```

Comment:
You should be able to do @subql/.... here, sometimes you need to restart ts server.

Reply:
In that case, the `yarn test` should run after `yarn build`.
The reason for this change is that we need to run the test locally with the changes before make the PR other than rely on the remote server to test the cases